app/views/submission_mailer/sentMsg.text.erb

```diff
@@ -0,0 +1,13 @@
+Szanowna Pani/ Szanowny Panie <%= @submission.person.surname %>
```

Comment:
zmienić na `send_status` (choć teraz myślę, że lepiej byłoby to nazwać `send_decision`).